dev-docs/modules/userId.md

```diff
@@ -258,6 +258,7 @@ Bidders that want to support the User ID module in Prebid.js need to update thei
 | Unified ID | Trade Desk | tdid | adserver.org | "1111" |
 | ConnectID | Yahoo | connectId | yahoo.com | {"connectId": "72d04af6..."} |
 | FreePass ID | FreePass | freepassId | | "1111" |
+| UtiqMtp ID | Utiq | utiqMtpId | utiq-mtp.com | "1111" |
```

Comment:
shouldn't the original `utiq` id module be listed in here as well?

Reply:
I know there was another Pull Request on the Utiq original module, which had breaking change and is planned for next major version release. This may be adressed as part of this PR. I would rather keep this separated. Happy to open another PR for this if this isn't already covered.